docs/presets/jetpack.md

```diff
@@ -0,0 +1,26 @@
+[Return to Presets](./README.md#jetpack)
+
+# Jetpack Preset
+
+This is a pseudo minimalist preset inspired by the [geometry](https://github.com/geometry-zsh/geometry) and [spaceship](https://github.com/spaceship-prompt/spaceship-prompt) prompts.
+
+> Jetpack uses the terminal's color theme.
+
+![Screenshot of Jetpack preset](/presets/img/jetpack.png)
+
+### Prerequisite
+
+- Requires a shell with [`right-prompt`](https://starship.rs/advanced-config/#enable-right-prompt) support.
+- [Jetbrains Mono](https://www.jetbrains.com/lp/mono/) is recommended.
+
+### Manual Configuration
+
+[Click to download TOML](/presets/toml/jetpack.toml)
+
+### Configuration
+
+```sh
+starship preset jetpack -o ~/.config/starship.toml
+```
+
```

Comment:
To be in line with the documentation of the other presets, I suggest:

```suggestion
### Configuration

```sh
starship preset jetpack -o ~/.config/starship.toml
```

```suggestion
[Click to download TOML](/presets/toml/jetpack.toml)
```

If the suggestions don't render correctly: put the TOML link just below the command line section.